modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -208,22 +208,18 @@ If you do not provide Python 3.6, the installation of the [package]``spacecmd``
 
 
 .Procedure: Adding Base Media to Custom Channels
-. On the {productname} Server, at the command prompt, as root, copy the base media image to the [path]``/tmp/`` directory.
-. Create a directory to contain the media content.
-    Replace [command]``<os_name>`` with either ``sll7``, ``sll8``, or ``sll9``:
+. Copy the base media image to the {productname} Server container host to the [path]``/tmp/`` directory.
+. To get the base media image into the {productname} Server container, use
```

Comment:
```suggestion
. To get the base media image into the {productname} Server container, use:
```

Reply:
I had this initially, but removed it before the creation of the PR. Added again. Thank you.